web/styles/portico/svg_icons.css

```diff
@@ -0,0 +1,105 @@
+}
+
+.client-logos__logo_layershift {
```

Comment:
Can we use `web/images/landing-page/logos/` for these, rather than having PNG files embedded in our CSS files?

(I'm pretty sure it was this way in the export from the design tool for no particular reason).

Reply:
Can you start a thread in #frontend about this? It seems unlikely to me that this is a way we want to do things. I can imagine configuring webpack to embed the `.png` files into a CSS file if that's what we want, but it doesn't make sense to me to take the source files (the third party's logo) and inline it into our CSS as the source format.

Reply:
The only reason I did it because we want to have svg images of logos be loaded before any animation. I couldn't find svg version for those png logos, so I made them this ugly way.
If we can get svgs for those that would be great, if we want to store them as files - ok, but lets make sure browser download them before animation

Reply:
OK this is not a blocker but I added a comment documenting what's weird about this file so we know what the story is.